pkg/koordlet/statesinformer/states_noderesourcetopology.go

```diff
@@ -87,20 +88,23 @@ func (s *statesInformer) calGuaranteedCpu(usedCPUs map[int32]*extension.CPUInfo,
 	if err != nil {
 		return nil, err
 	}
-	podAllocs := []extension.PodCPUAlloc{}
-	for pod := range checkPoint.Entries {
+	var podAllocs []extension.PodCPUAlloc
+	for podUID := range checkPoint.Entries {
```

Comment:
kubelet checkpoint will contains:
1、Guranteed pods
2、LSE/LSR pods

LSE/LSR pods will appear in checkpoint as they are also guranteed QoS. The koordlet must exclude these Pods, otherwise it will cause the scheduler to calculate errors.

Reply:
updated.